### PR TITLE
mongoose model

### PR DESCRIPTION
**Pull request by someone-1**
added 2 models 
- `/model/club.js` contains info of club
- `/model/user.js` contains info of user

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,10 +1,32 @@
...
+//         'location' : 'String'
+//     }).save().then((newUser) => {
+//         console.log('created new user ' , newUser)
+//         res.send(newUser)
+//     })
+// })
+
 const PORT = process.env.PORT || 8080;
-app.listen(PORT , function(){
+app.listen(PORT , ()=>{
```
Comment (awebdeveloper) [inline index.js]: plz use express generator. the port n all should be in a diff file

File: model/club.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+const clubSchema = new mongoose.Schema({
+    'name': String,
+    'club-id' : String,
+    'city': String,
+    'more-info' : String,
+    'location' : String
+})
+
+const club = mongoose.Model('club' , clubSchema);
+module.exports = club;
```
Comment (awebdeveloper) [inline model/club.js]: missing trailing enter

File: model/user.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
```
Comment (awebdeveloper) [inline model/user.js]: move this to a schema file inside model

File: model/club.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
```
Comment (awebdeveloper) [inline model/club.js]: move this to a schema folder inside model

File: model/club.js [hunk 5]
```diff
@@ -1,11 +1,4 @@
 const mongoose = require('mongoose');
-const clubSchema = new mongoose.Schema({
-    'name': String,
-    'club-id' : String,
-    'city': String,
-    'more-info' : String,
-    'location' : String
-})
-
+const clubSchema = require('./schema/clubSchema')
```
Comment (awebdeveloper) [inline model/club.js]: use -(hyphen) case for file names

File: model/schema/userSchema.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+const userSchema = new mongoose.Schema({
+    'user-name': String,
+    'google-id' : String,
+    'city': String,
+    'profile-pic-url' : String,
+    'club-id' : Array,
+    'location' : String
+})
+
+module.exports = userSchema;
```
Comment (awebdeveloper) [inline model/schema/userSchema.js]: missing trialling enter

File: model/schema/clubSchema.js [hunk 7]
```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose');
+const clubSchema = new mongoose.Schema({
+    'name': String,
+    'club-id' : String,
+    'city': String,
+    'more-info' : String,
+    'location' : String
+})
+
+module.exports = clubSchema;
```
Comment (awebdeveloper) [inline model/schema/clubSchema.js]: missing trailing enter

File: model/schema/userSchema.js [hunk 8]
```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
```
Comment (awebdeveloper) [inline model/schema/userSchema.js]: - case for file name

File: model/schema/clubSchema.js [hunk 9]
```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose');
```
Comment (awebdeveloper) [inline model/schema/clubSchema.js]: use - case for file name

Comment (awebdeveloper) [reply model/schema/clubSchema.js]: see what jquery does https://stackoverflow.com/questions/7273316/what-is-the-javascript-filename-naming-convention

**Comment by someone-1** [comment]
i am not sure if what i did to fix `trailing enter` was done the right way.
i added a new line with a space in it

**Pull request merged**